src/repr/src/row/collection.rs

```diff
@@ -33,6 +35,8 @@ pub struct RowCollection {
     encoded: Bytes,
     /// Metadata about an individual Row in the blob.
     metadata: Vec<EncodedRowMetadata>,
+    /// Start of sorted runs of rows in rows.
+    fingers: Vec<usize>,
```

Comment:
The documentation here confused me. This field actually stores the indexes of the _ends_ of sorted runs, right? Is there a reason for that? It does feel like storing the start indexes would be more natural.

I know you said the PR lacks documentation, so if you still planned to adjust it here then nvm!